product-matrix.json

```diff
@@ -23,23 +23,23 @@
       "untilBuild": "191.*"
     },
     {
-      "comments": "Android Studio 3.6 Beta 1",
+      "comments": "Android Studio 3.6 Beta 2",
```

Comment:
Does this now support `2019.2.3`?

Reply:
Sorry, but I don't know if beta 2 is based on 2019.2.3 or not. I think it is.